.circleci/config.yml

```diff
@@ -203,11 +203,11 @@ jobs:
             #: Lookup cache based on the default branch and a timestamp. Allows
             #: to use cache from the very first build on the day (sanitized
             #: database dump, for example).
-            - v1-db8-{{ checksum "/tmp/DB_CACHE_BRANCH" }}-{{ checksum "/tmp/DB_CACHE_FALLBACK" }}-{{ checksum "/tmp/DB_CACHE_TIMESTAMP" }}
+            - v5-db9-{{ checksum "/tmp/DB_CACHE_BRANCH" }}-{{ checksum "/tmp/DB_CACHE_FALLBACK" }}-{{ checksum "/tmp/DB_CACHE_TIMESTAMP" }}
```

Comment:
here and below - had to increment the cache key to invalidate old DB docker images (if not invalidated - old images will be used in tests)